frontend/src/container/LogControls/config.ts

```diff
@@ -0,0 +1 @@
+export const ITEMS_PER_PAGE_OPTIONS = [25, 50, 100, 200];
```

Comment:
This could be placed in the same component itself, unless we will use this elsewhere

Reply:
i guess this is more relevant to the component itself... but don't want to put all the stuff in one place